ODCRequest/js/RequestForm.js

```diff
@@ -209,44 +209,46 @@ define(["dojo/_base/declare",
           content: error,
           style: "width: 400px"
         });
-        alert_dialog.show()
+        alert_dialog.show();
       },
 
       checkDatePosition: function () {
+        let parent = this.getParent().getParent();
         const date_pairs = {
           'time_start': 'time_end',
           'analysis_time_start': 'analysis_time_end',
           'baseline_time_start': 'baseline_time_end'
-        }
+        };
 
+        let date_objects = parent.date_objects
```

Comment:
I don't understand the purpose of this local variable, do you want to have a share screen session for this?

Reply:
I probably just could have continued using parent.date_objects, just thought it looked more consistent in terms of naming. Any tips you have I'd greatly appreciate definitely!